### PR TITLE
17596 - support releasing Che docs 

### PR DESCRIPTION
**Pull request by nickboldt**
17596 - remove logic for maven version bumping; instead just use a VERSION file like in other repos;
17596 - instead of defaulting to PR generation, try to push directly;
17596 - instead of pushing to release branch, just tag the branch
17596 - bump master to 7.17.1-SNAPSHOT in prep for Che 7.17.1 release

Change-Id: Icdb910cb28583b6cd61a294e23286287d8066d71
Signed-off-by: nickboldt <nboldt@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?

* remove logic for maven version bumping; instead just use a VERSION file like in other repos;
*  instead of defaulting to PR generation, try to push directly;
*  instead of pushing to release branch, just tag the branch
*  bump master to 7.17.1-SNAPSHOT in prep for Che 7.17.1 release

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/17596

### Specify the version of the product this PR applies to. 

master + 7.17.1

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+7.17.1-SNAPSHOT
```
Comment (benoitf) [inline VERSION:1]: should be 7.18.0-SNAPSHOT as 7.17 is in a branch

Comment (benoitf) [reply VERSION:1]: cherry-pick into 7.17.x branch ?

**Review by MichalMaler — APPROVED** [review]

File: VERSION (hunk 1 above)
Comment (nickboldt) [reply VERSION:1]: doesn't really matter as the value is meaningless, but if you WANT I can change it over. It'll be set to 7.18.1-SNAPSHOT in 2 weeks when we release 7.18.0

Comment (nickboldt) [reply VERSION:1]: Followup in https://github.com/eclipse/che-docs/pull/1463

**Pull request merged**